distribution/docs/src/main/resources/content/_coreConcepts/map-standards.adoc

```diff
@@ -8,39 +8,51 @@
 
 ${catalog-ui} includes capabilities to support custom map layer providers as well as support for several popular map layer providers.
 
+Some provider types are currently only supported by the https://openlayers.org[2D Openlayers] map and some only by the https://cesiumjs.org[3D Cesium] map.
+
 .Map Formats Included in ${ddf-branding}
-[cols="1,1" options="header"]
+[cols="1,1,1" options="header"]
 |===
 
 |Format
-|Status
+|2D Documentation
+|3D Documentation
 
-|https://cesiumjs.org/Cesium/Build/Documentation/createOpenStreetMapImageryProvider.html[Open Street Map]
-|Supported
+|Open Street Map
+|https://openlayers.org/en/latest/apidoc/ol.source.OSM.html[Open Layers]
+|https://cesiumjs.org/Cesium/Build/Documentation/createOpenStreetMapImageryProvider.html[Cesium]
```

Comment:
I think these links OpenLayers and Cesium links should point to the documentation for the current version of cesium that is used in DDF, not just the documentation for the latest version.

Reply:
I agree.

Reply:
Looks like catalog-ui-search uses openlayers 3.20.1 https://github.com/codice/ddf/blob/4e27c8f96b0119dceab9de21afe6dc6df24f9571/catalog/ui/catalog-ui-search/package.json#L96 (or maybe 3.16.0? https://github.com/codice/ddf/blob/e8163ed0493711cd2724e1657e14046f95209c21/catalog/ui/search-ui/standard/pom.xml#L266-L270) and cesium 1.34 https://github.com/codice/ddf/blob/4e27c8f96b0119dceab9de21afe6dc6df24f9571/catalog/ui/catalog-ui-search/package.json#L79  (or maybe 1.22.0? https://github.com/codice/ddf/blob/e8163ed0493711cd2724e1657e14046f95209c21/catalog/ui/search-ui/standard/pom.xml#L151-L155).

So the links in this file and https://github.com/codice/ddf/blob/a45417ec2878dc76dcf5dfaf621887fb5ead3622/catalog/admin/module/catalog-admin-module-maplayers/src/main/webapp/map-layers/options.json should be updated to point to these specific versions. I'm having trouble finding a direct link to the javadoc for these not-latest versions. I think updating these to say `See the OpenLayers 3.20.1 documentation at http://openlayers.org/` and `See the Cesium 1.34 documentation at https://cesiumjs.org` would be better than linking directly to the latest docs.

Reply:
Also, I think OpenLayers might be one word: http://openlayers.org/en/latest/doc/tutorials/introduction.html.

Reply:
Shouldn't those pom and package.json versions match up, and then you can just property it into the docs as well?

Reply:
That's what I'm hoping to do

Reply:
I think the package.json version should be the one used by yarn/npm/whatever we use to actually fetch the dependencies on install. Not sure what the webjar dependencies are necessarily for, though.

Reply:
I would also go with the package.json version

Reply:
Should the others be a) made to match? b) deleted entirely? c) left alone?

Reply:
looking at these again, it appears the difference is one version is intrigue and the other is the standard search ui.